.github/workflows/typescript-nudge.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  pull_request:
+    paths:
+      - '**.js'
+      - '**.jsx'
```

Comment:
We don't use jsx but we have cjs and mjs so I would adapt to:

```
- '**.js'
- '**.cjs'
- '**.mjs'
- '**.ts'
- '**.mts'
- '**.cts'

Reply:
Good to know! That makes total sense 👍🏻